From ce1fef1e1792003f398bdc4055dff39ad8624ba9 Mon Sep 17 00:00:00 2001
From: Shilin Victor <chrono.monochrome@gmail.com>
Date: Thu, 20 Sep 2018 09:47:43 +0300
Subject: [PATCH 57/67] Revert "OMXNodeInstance: use a lock around
 OMX::freeNode"

This commit breaks the audio and/or video playback on STE devices.

Change-Id: I6ed46c6061bc5437408e42e6412bf48c178f910e
---
 media/libstagefright/omx/OMXNodeInstance.cpp | 2 --
 1 file changed, 2 deletions(-)

diff --git a/media/libstagefright/omx/OMXNodeInstance.cpp b/media/libstagefright/omx/OMXNodeInstance.cpp
index 8a698e3aa..aa5ca3f09 100644
--- a/media/libstagefright/omx/OMXNodeInstance.cpp
+++ b/media/libstagefright/omx/OMXNodeInstance.cpp
@@ -496,8 +496,6 @@ status_t OMXNodeInstance::freeNode() {
             break;
     }
 
-    Mutex::Autolock _l(mLock);
-
     status_t err = mOwner->freeNode(this);
 
     mDispatcher.clear();
-- 
2.11.0

